### DISCUSSION TITLE
Removing tracked languages from my data?

### DISCUSSION THREAD

**Discussion by BenjaminLincoln** [muety/wakapi#439] [Q&A]
Hi, I'm sorry if I've just missed this feature but is there a way to remove languages that got tracked by wakapi through the .dev website?

**Comment by muety** [comment]
Hi @CuriousFullStacks, I feat there is currently no way to do that. But could you please specify your concern a bit further? Would you wanna remove all coding data that was coded in a specific language? Only get rid of the language tag? I think it would be helpful to understand your particular use case a bit better to then perhaps create a new feature request :-).

**Reply by BenjaminLincoln** [reply]
I've noticed that clicking on a .md file can leave an INI language stuck on your dashboard with no option to remove it, and I'd prefer to only show languages I spend time with for more than 30 seconds or use for more than config for a few minutes at a time. Whether that would be removing coding data or the language tag wouldn't be too important to me. I do think it could be a useful feature for people who only want to show what they actually spend time working on over every single file that gets opened in their IDE.

**Reply by muety** [reply]
Wait, if you code on a `.md` file, it's being display as INI? That's not how it's supposed to be :thinking:.

But apart from that, thanks for providing me with a better understanding of your use case. A while ago, I had the idea of kind of a _data explorer_ feature, where you can visually inspect the raw, underlying data, i.e. durations or even individual heartbeats, on a timeline graph. Given that, you could then also easily delete single data points or a range of such. I'll create a ticket for this, but will be quite a big new feature (requires a whole new view), so please don't expect it to be realized very soon.

Another potentially useful option for you could be to ignore certain file endings (such an `.md`) in your WakaTime CLI config, see [here](https://github.com/wakatime/wakatime-cli/blob/develop/USAGE.md).

Hope that helps.

**Reply by BenjaminLincoln** [reply]
Thanks for taking the time to reply, for the time being I was able to hide INI from the github stats card I'm using on my profile so no issue anymore really. I'm sure the feature would still be great to have though.
